README.md

```diff
@@ -27,6 +27,17 @@ Each scenario uses an AWS CloudFormation template to  deploy your game, creating
 * If you cloned the repo you will also need to download the following items from [Amazon GameLift's Getting Started](https://aws.amazon.com/gamelift/getting-started/). Otherwise, if you downloaded the zip file from [the GitHub Releases page](https://github.com/aws/amazon-gamelift-plugin-unreal/releases), you can skip this step.
     * C++ Server SDK Plugin for Unreal
     * C++ Server SDK
+* If you cloned the repo instead of downloading the zip from [the GitHub Releases page](https://github.com/aws/amazon-gamelift-plugin-unreal/releases), copy the source code for C++ Server SDK Plugin for Unreal into the folder `amazon-gamelift-plugin-unreal-release-<version>/amazon-gamelift-plugin-unreal/GameLiftPlugin/Source/GameLiftServer/`. The `REPLACEME.md` inside the directory details what the file structure should look like. Delete the `GameLiftServerSDK.uplugin` from the same directory. Then make the following code change to `amazon-gamelift-plugin-unreal/GameLiftPlugin/Source/GameLiftServer/Source/GameLiftServerSDK/Private/GameLiftServerSDK.cpp`. Change line 30 and 31 from
+```
+FString BaseDir = IPluginManager::Get().FindPlugin("GameLiftServerSDK")->GetBaseDir();
+const FString SDKDir = FPaths::Combine(*BaseDir, TEXT("ThirdParty"), TEXT("GameLiftServerSDK"));
+```
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;to
```

Comment:
is it intentional to have &nbsp here ?

Reply:
yes it's for formatting so that the "to" and "If you downloaded the zip file from..." sentence align with the bullet point in markdown- 
<img width="1396" alt="Screenshot 2024-03-07 at 12 33 17 PM" src="https://github.com/aws/amazon-gamelift-plugin-unreal/assets/138809155/9229bea6-a020-4a67-8011-46b05d622564">